test/ctia/http/middleware/ratelimit_test.clj

```diff
@@ -122,7 +122,8 @@
                (dotimes [_ 4] (call app))
                (let [response (call app)]
                  (is (= 429 (:status response)))
-                 (is (= "3600" (get-in response [:headers "Retry-After"])))
+                 (is (#{"3599" "3600"} ;; avoid flakyness on slow test server.
```

Comment:
With a range from 3593 to 3600 you'd guard against a variety of "acceptable slowness" (e.g. slow Travis, GC pauses, test fixtures, etc) without compromising much the accuracy of the test.

Reply:
Yes I agree. I just observed  3595 on my local env.
Due to some kind of OCD I was about to push 3590, any special explanation for 3593 :-)?

Reply:
Ok nice, I still in the range with 10 and my OCD that pushes me to use a "round" value :-)